sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -1417,7 +1417,12 @@ def _add_random_shard(element):
       value = element[1]
       return ((key, random.randint(0, DEFAULT_SHARDS_PER_DESTINATION)), value)
 
-    def _to_hashable_table_ref(table_ref_elem_kv):
+    V = TypeVar('V')
+    TABLE_REF_TYPE = TypeVar('TABLE_REF_TYPE')
+
+    def _to_hashable_table_ref(
```

Comment:
This could be put in bigquery_tools.py to avoid code repetition.

Reply:
Good idea. I moved it to bigquery_tools.py